.github/workflows/backend_sumo_prod.yml

```
@@ -0,0 +1,63 @@
+name: integration
+
+on:
+  push:
+  pull_request:
+    branches:
+      - main
+  release:
+    types:
+      - published
+  # workflow_dispatch:
+  # schedule:
+  #   - cron: "48 4 * * *"
```

```suggestion
  push:
  workflow_dispatch:
  schedule:
    - cron: "0 0 * * *"
```
Due to the secrets not working for incoming PRs (from forks), we for now limit to only runs initiated from the repo itself (push to branch in official repo, workflow dispatch and schedule).